teamapps-ux/src/main/java/org/teamapps/ux/session/SessionContext.java

```diff
@@ -152,14 +160,14 @@ public SessionContext(QualifiedUiSessionId sessionId,
 	}
 
 	/**
-	 * @return the SessionContext which the thread, where this method is executed in, is bound to
+	 * @return the SessionContext which the thread is bound to where this method is executed in
```

Comment:
@return the SessionContext which the current thread is bound to

The rest of the sentence is redundant